cmd/skopeo/sync.go

```diff
@@ -296,21 +388,23 @@ func syncHandler(c *cli.Context) (retErr error) {
 			SourceCtx:             toCopy.SourceCtx,
 		}
 
-		fmt.Printf("Processing image %d/%d\n",
-			counter+1,
-			len(toCopy.SourceReferences))
-		destRef, err := buildFinalDestination(toCopy.RootSrcRef, ref, c.Args()[0])
-		if err != nil {
-			return err
-		}
-		logrus.WithFields(logrus.Fields{
-			"source":      ref,
-			"destination": destRef,
-		}).Info("Copy started")
+		for counter, ref := range toCopy.SourceReferences {
+			fmt.Printf("Processing image %d/%d\n",
+				counter+1,
+				len(toCopy.SourceReferences))
```

Comment:
I would extend the message given to the user to make him aware of the source registry that we are copying from.

The new code iterates over a list of different sources (`docker.io`, `quay.io`,...) and shows the progress of images to be copied. Without knowing the name of the source registry it's a bit weird to look at the stdout.

Reply:
OK. I mostly used logrus.Info*, while you mostly used fmt.Print*. Should we converge to a common one or how to tell when one is better than the other?